.travis/install-dependencies.sh

```diff
@@ -20,15 +20,17 @@ fi
 if [[ "$PYTHON" == "2.7" ]] && [[ "$platform" == "linux" ]]; then
   sudo apt-get update
   sudo apt-get install -y cmake build-essential autoconf curl libtool python-dev python-numpy python-pip libboost-all-dev unzip
-  sudo pip install cloudpickle funcsigs colorama psutil redis tensorflow
+  sudo pip install cloudpickle funcsigs colorama psutil redis
+  sudo pip install tensorflow==0.12.0
```

Comment:
Yeah, it's already out and it broke `python test/tensorflow_test.py`, hence the downgrade :(